wire-library/wire-test-utils/src/main/java/com/squareup/wire/schema/JvmGenerator.kt

```diff
@@ -17,94 +17,44 @@ package com.squareup.wire.schema
 
 import com.squareup.javapoet.JavaFile
 import com.squareup.kotlinpoet.FileSpec
-import com.squareup.wire.ProtoAdapter
 import com.squareup.wire.java.JavaGenerator
 import com.squareup.wire.java.Profile
+import com.squareup.wire.java.internal.ProfileParser
 import com.squareup.wire.kotlin.KotlinGenerator
 import com.squareup.wire.kotlin.RpcCallStyle
 import com.squareup.wire.kotlin.RpcRole
-import okio.Path.Companion.toPath
-import okio.buffer
-import okio.fakefilesystem.FakeFileSystem
-import okio.source
-import java.io.File
 import java.io.IOException
 
-/**
- * Builds a repository of `.proto` and `.wire` files to create schemas, profiles, and adapters for
- * testing.
- */
-class RepoBuilder {
-  private val fs = FakeFileSystem()
-  private val root = "/source".toPath()
-  private val schemaLoader = SchemaLoader(fs)
-  private var schema: Schema? = null
+// TODO(Benoit) Maybe just have static method? and pass the profile in params.
+/** Helper class to run Java and Kotlin code generation. */
+class JvmGenerator(
+  private val schema: Schema
+) {
+  private val profiles = mutableMapOf<String, Profile>()
```

Comment:
Because schema building and code generation are abstracted away from each other, I changed the API around setting up profiles. Instead of write a file that we'd parse later, I decided to parse them directly and store them in memory.